debezium-server/debezium-server-bom/pom.xml

```diff
@@ -16,7 +16,7 @@
         <version.pubsub>5.1.0</version.pubsub>
         <version.pulsar>2.5.2</version.pulsar>
         <version.eventhubs>5.1.1</version.eventhubs>
-        <version.jedis>3.5.2</version.jedis>
+        <version.jedis>4.1.1</version.jedis>
```

Comment:
Slightly related: we want to use the Quarkus Redis client going forward (see [DBZ-4724](https://issues.redhat.com/browse/DBZ-4724)); just a heads-up, doesn't have to stop us from doing this update. // CC @spicy-sauce

Reply:
I agree, let's leave it as it was - I intend to replace it Quarkus Redis.

Reply:
May be `3.8.0` is a _safer_ option?